lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -6,6 +6,15 @@ module Contrib
     # Rack module includes middlewares that are required to trace any framework
     # and application built on top of Rack.
     module Rack
+      # RACK headers to test when doing distributed tracing.
+      # They are slightly different from real headers as Rack uppercases everything
+
+      # Header used to transmit the trace ID.
+      HTTP_HEADER_TRACE_ID = 'HTTP_X_DATADOG_TRACE_ID'.freeze
+
+      # Header used to transmit the parent ID.
+      HTTP_HEADER_PARENT_ID = 'HTTP_X_DATADOG_PARENT_ID'.freeze
```

Comment:
Good feedback. Indeed configurable entries would ease up your migration process and allow fine-tuning. OTOH these should really be the same across all languages, implementations, etc. At some point it's a bit like having standard HTTP headers like `WWW-Authenticate` be tuneable (randomly picked from https://tools.ietf.org/html/rfc2617 but you get the idea). Plus the fact in some areas you want them uppercased, and in some not. I fear this could become complicated. But again, will try and see if I can find a way that makes it easy without introducing any complexity.